pkg/proxy/ipvs/netlink_linux.go

```diff
@@ -105,6 +105,23 @@ func (h *netlinkHandle) DeleteDummyDevice(devName string) error {
 	return h.LinkDel(dummy)
 }
 
+// ListBindAddress will list all IP addresses which are bound in a given interface
+func (h *netlinkHandle) ListBindAddress(devName string) ([]string, error) {
+	dev, err := h.LinkByName(devName)
+	if err != nil {
+		return nil, fmt.Errorf("error get interface: %s, err: %v", devName, err)
```

Comment:
Error get ...

Reply:
this error log is `return value`, will not print out directly, so I think `error` should keep in `lower-case` word. Just like other function's log format in this file , `EnsureAddressBind`, `UnbindAddress` ...

Reply:
this error log is `return value`, will not print out directly, so I think `error` should keep in `lower-case` word. Just like other function's log format in this file , `EnsureAddressBind`, `UnbindAddress` ...